tests/src/RunBenchmarks/build.cmd

```diff
@@ -0,0 +1,10 @@
+rem *** Build for Desktop CLR
+
+mkdir bin\Debug\desktop
+csc /define:DESKTOP /nologo /debug /target:exe /out:bin\Debug\desktop\RunBenchmarks.exe RunBenchmarks.cs 
+
+rem *** Build for Core CLR
+
+dotnet restore
+dotnet compile
+
```

Comment:
Same comment as before: scripts should be as well-commented as source code.

Should you add setlocal?

Should the Desktop/CoreCLR builds be in different scripts?
Should you check for and report errors?
Should you report where the results were generated?